### DISCUSSION TITLE
How does this script compare to the one made by sycnex ?

### DISCUSSION THREAD

**Discussion by uczpre** [LeDragoX/Win-Debloat-Tools#175] [General]
https://github.com/Sycnex/Windows10Debloater
this one i mean.

**Comment by LeDragoX** [comment]
I don't know because there are a lot of changes which I will not analyse, but their last commit/update were from **Jun 27, 2021, 12:39 AM GMT-3**, and the repository is now archived, so it's not maintained by anyone.
